osu.Game/Screens/Edit/Timing/MetronomeDisplay.cs

```diff
@@ -268,19 +265,25 @@ protected override void LoadComplete()
 
         private void updateBpmText()
         {
-            int intPart = (int)interpolatedBpm.Value;
-
-            bpmText.Text = intPart.ToLocalisableString();
+            string text = interpolatedBpm.Value.ToString("N2");
+            int? breakPoint = null;
 
-            // While interpolating between two integer values, showing the decimal places would look a bit odd
-            // so rounding is applied until we're close to the final value.
-            int decimalPlaces = FormatUtils.FindPrecision((decimal)effectiveBpm);
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (!char.IsDigit(text[i]))
+                    breakPoint = i;
+            }
 
-            if (decimalPlaces > 0)
+            if (breakPoint != null)
             {
-                bool reachedFinalNumber = intPart == (int)effectiveBpm;
+                bool reachedFinalNumber = (int)interpolatedBpm.Value == (int)effectiveBpm;
 
-                bpmText.AddText((effectiveBpm % 1).ToLocalisableString("." + new string('0', decimalPlaces)), cp => cp.Alpha = reachedFinalNumber ? 0.5f : 0.1f);
+                bpmText.Text = text.Substring(0, breakPoint.Value);
+                bpmText.AddText(text.Substring(breakPoint.Value), cp => cp.Alpha = reachedFinalNumber ? 0.5f : 0.2f);
+            }
+            else
+            {
+                bpmText.Text = text;
```

Comment:
Two things:

- This makes it so that the metronome *always* shows 2 decimals even if the value is a whole number - not sure if that was intended
- The other thing may be borderline out of scope because it's not really changed here but I find the alpha behaviour here very strange because it's only apparently using the `0.2f` alpha when interpolating towards a whole number rather than just interpolating period?

https://github.com/user-attachments/assets/f0cf9fdd-8bcf-4377-bc7d-4c611112e908